### PR TITLE
add get_historical_stats_for_account endpoint

### PR DESCRIPTION
**Pull request by stas-bool**
[Doc](https://bungie-net.github.io/multi/operation_get_Destiny2-GetHistoricalStatsForAccount.html#operation_get_Destiny2-GetHistoricalStatsForAccount)

### CODE REVIEW

File: pydest/api.py [hunk 1]
```diff
@@ -252,6 +252,22 @@ async def get_historical_stats(self, membership_type, membership_id, character_i
         url = url.format(membership_type, membership_id, character_id, ','.join([str(i) for i in groups]), ','.join([str(i) for i in modes]))
         return await self._get_request(url)
 
+    async def get_historical_stats_for_account(self, membership_type, membership_id, groups=[]):
```
Comment (jgayfer) [inline pydest/api.py:255]: Can we add a test and update README.md as well please?

Comment (stas-bool) [reply pydest/api.py:255]: Done

Comment (jgayfer) [reply pydest/api.py:255]: Thanks! Will take it for a spin tomorrow.

Comment (jgayfer) [reply pydest/api.py:255]: "Tomorrow" turned into a couple months, but I've pushed this change to PyPi under the 0.4.0 release :)

Thanks again!

**Pull request merged**